packages/playwright/src/generators/init/generator.spec.ts

```diff
@@ -168,11 +202,134 @@ describe('playwright generator', () => {
         expect(gitIgnoreFile).toContain('**/playwright-report');
         expect(gitIgnoreFile).toContain('**/playwright/.cache');
 
+        const taskctlYAML = YAML.parse(appTree.read('taskctl.yaml', 'utf8'));
+        expect(taskctlYAML.pipelines.dev).toContainEqual({
+            task: 'e2e:ci',
+            depends_on: 'build',
+        });
+        expect(taskctlYAML.pipelines.fe).toContainEqual({
+            task: 'e2e:ci',
+            depends_on: 'build',
+        });
+        expect(taskctlYAML.pipelines.nonprod).toContainEqual({
+            task: 'e2e:ci',
+            depends_on: 'test:ci',
+        });
+        expect(taskctlYAML.pipelines.prod).toContainEqual({
+            task: 'e2e:ci',
+            depends_on: 'test:ci',
+        });
+
         // Add infra tasks
         const projectJson = readJson(
             appTree,
             joinPathFragments(projectNameE2E, 'project.json'),
         );
         expect(projectJson.targets.e2e).toBeTruthy();
     }, 100_000);
+
+    it('should alter azure devops stages file that is generated by workspace', async () => {
+        appTree.write(
+            'build/azDevOps/azuredevops-stages.yaml',
+            `parameters:
```

Comment:
Please move this line to the next one so the indentation remains valid for a YAML file.